pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+
+	export let title = 'Stock';
+	export let currentLocation: string;
+
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+</script>
+
+<header class="w-full bg-gray-900 px-[70px]">
+	<div class="flex h-16 items-center border-b border-sky-900 ">
+		<div class="mr-4">
+			<Logo />
+		</div>
+
+		<nav class="flex space-x-4">
+			{#each links as { label, href }}
+				<a
+					{href}
+					class="
+							rounded-md px-3 py-2 text-sm
+							{currentLocation === href ? 'bg-teal-500 text-gray-900' : 'text-white hover:text-teal-500'}"
+					aria-current={currentLocation === href ? 'page' : 'false'}
+				>
+					<span>{label}</span>
+				</a>
+			{/each}
+		</nav>
+	</div>
+	<div class="flex h-24 items-center justify-between">
+		<h2 class="text-3xl font-bold leading-7 text-white">
+			{title}
+		</h2>
+
+		<div class="flex">
+			<slot name="actionButton" />
+		</div>
```

Comment:
Can we simply render the `slot` here without the `flex` container....it already is in one flex container (the second row), so I don't see the benefit of `<div class="flex" >` wrapping this